tests/boot/boot_from_pxe.pm

```diff
@@ -217,6 +231,10 @@ sub run {
                 send_key "ret";
                 assert_screen $ssh_vnc_tag, $ssh_vnc_wait_time;
             }
+            elsif (match_has_tag('agama-installer-live-root')) {
+                record_info('PXE boot successfully, Agama installer start');
+                return;
+            }
```

Comment:
I would suggest move this code block to line 156 :) agama boot so that we have simple logic for agama install only